src/olympia/blocklist/tests/test_admin.py

```diff
@@ -1060,7 +1060,58 @@ def test_cannot_approve_with_only_block_create_permission(self):
         assert mbs.reason != 'a reason'
 
     def test_cannot_reject_with_only_block_create_permission(self):
```

Comment:
Should the test be renamed to indicate that it is also testing that a user with only `Blocklist:Create` *can* reject their own submission? If that is too much of a mouthful, should it be made into two different tests?

Reply:
https://github.com/mozilla/addons-server/pull/13622/commits/91e55bf467f9aa70128590fd5f99c65dd28f6f15